dist/js/entity/in_memory.d.ts

```diff
@@ -49,7 +49,7 @@ export declare class InMemoryEntity {
      * @summary Clone this entity
      */
     clone(extraContext?: object): this;
-    static validateData(data: AnyObject, clean?: boolean): AnyObject;
+    static validateData(data: AnyObject, clean?: boolean, jsonSchema?: import("json-schema").JSONSchema7 | undefined): AnyObject;
```

Comment:
@k0stik - is this a common convention to import inline here or should we move this to top?

Reply:
Oh, I see - this is dist